_data/buildx/docker_buildx_bake.yaml

```diff
@@ -5,7 +5,7 @@ long: |-
     Bake is a high-level build command. Each specified target will run in parallel
     as part of the build.
 
-    Read [High-level build options with Bake](/build/bake/)
+    Read [High-level build options with Bake](/build/customize/bake/)
```

Comment:
@dvdksn We should update this link on buildx repo to `/build/bake/`

Reply:
Oooh I see